.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "10.15.3"
```

Comment:
I wonder based on what we know what is the minimum version of node? React?

Reply:
I couldn't use latest version because I was getting errors from jest. This might be fixed in the future, but as for now we'll use LTS version of node